scripts/flattener.py

```diff
@@ -1189,7 +1194,8 @@ def main(mfinal_id):
 				collapsed_adata = ad.concat([collapsed_adata, collapsed_row], axis=1, join='outer', merge='first')
 			del(collapsed_row)
 			gc.collect()
-		print(collapsed_adata)
+		results = "/Users/jychien/Lattice-Data/lattice-tools/scripts/collapsed_adata.h5ad"
```

Comment:
These two lines were for QC. I should have removed them.

Reply:
Lines 1197 and 1198

Reply:
Just to clarify, do you want me to delete 

results = "/Users/jychien/Lattice-Data/lattice-tools/scripts/collapsed_adata.h5
collapsed_adata.write(results, compression="gzip")
mfinal_adata = mfinal_adata[:, [i for i in mfinal_adata.var.index.to_list() if i not in all_drop]]

Reply:
The following 2 lines:
`results = "/Users/jychien/Lattice-Data/lattice-tools/scripts/collapsed_adata.h5ad"`
`collapsed_adata.write(results, compression="gzip")`